examples/flax_mnist.py

```diff
@@ -15,16 +15,32 @@
 """MNIST example with Flax and JAXopt."""
 
 from absl import app
+from absl import flags
+
 from flax import linen as nn
+
 import jax
 import jax.numpy as jnp
+
 from jaxopt import loss
 from jaxopt import OptaxSolver
+from jaxopt import PolyakSGD
 from jaxopt import tree_util
+
 import optax
+
 import tensorflow_datasets as tfds
 
 
+flags.DEFINE_float("l2_regul", 1e-4, "L2 regularization.")
+flags.DEFINE_float("learning_rate", 0.001, "Learning rate (ADAM).")
+flags.DEFINE_bool("manual_loop", False, "Whether to use a manual loop.")
+flags.DEFINE_float("max_step_size", 0.1, "Maximum step size (PolyakSGD).")
+flags.DEFINE_float("momentum", 0.9, "Momentum strength.")
+flags.DEFINE_string("solver", "adam", "Solver to use.")
```

Comment:
```suggestion
flags.DEFINE_enum("solver", "adam", ["adam", "sgd", "polyak-sgd"], "Solver to use.")
```

is safer and informs the reader what are the available options